### DISCUSSION TITLE
Is it normal to get “poor people” riots from medium insulae?

### DISCUSSION THREAD

**Discussion by Cimbali** [Keriew/augustus#957] [Q&A] [answered]
Thanks for this great content, love replaying a childhood game with so many improvements.

I was playing Londinium (Consul so mission 10 I believe) and I got criminality from medium insulae neighbourhoods. Also had actual riots from a small house neighbourhood that I hadn’t upgraded yet. Is this normal? Their dwellings aren’t really slums… Seemed a tad strange to me, also the criminality level for these insulae jumped from the lowest (beige) pillar to the highest red one instantly, which in turn blocked immigration (that’s probably normal).

For context, I had 8 small palaces on the same map and the neighbourhoods had stayed at the medium insulae for a while.

**Comment by crudelios** [comment, marked as answer]
With the new sentiment mechanics, yes it is normal.

Your pop now resents having higher level housing, so either you improve everyone's housing level (especially when you get to palaces) or you give them more entertainment/food types to compensate.

You'll get especially punished for small casa and below.

**Reply by Cimbali** [reply]
Thanks! They did have only 1 type of food (but tons of entertainment).

**Reply by crudelios** [reply]
Note that you can use the "sentiment" overlay to check the actual sentiment (red is bad, white is neutral, blue is good) and you can also right-click a house for a specific reason for their unhappiness.

Do note that you need to provide more entertainment than is required for that housing step for it to count as positive to the sentiment. The same with food, actually.

**Reply by Cimbali** [reply]
The sentiment overlay was all-white, even when criminality spiked, and the reason for unhappiness just stated they were fed up with living in such poor housing.
